pages/home/components/Intro.tsx

```diff
@@ -76,7 +76,9 @@ export const Intro: FC = ({}) => {
 				>
 					<Image src={group} style={{ width: "100%" }} alt="" />
```

Comment:
consider adding aria-labels for screen readers where-ever possible

Reply:
I did intentionally left some alt texts because some images have no beneficial for accessibility and SEO purpose. eg: memoji avatars in RibbonDecorations component. You can check that in [this article](https://dequeuniversity.com/rules/axe/4.4/image-alt?utm_source=lighthouse&utm_medium=devtools). Anyway, I've updated all the empty alt texts in home page.

To include the word `WalkingPal` in every alt text, in my opinion, is not very ideal. For instance, alt text of walking images in EasyStepsFooter component should be just a simple "a walking boy", "walking legs" etc. What do you think about this? However, I did add the word `WalkingPal` in alt texts as much as I can.

I don't see any aria-label warning in Home page. I think it is from Register page.

Reply:
> To include the word WalkingPal in every alt text, in my opinion, is not very ideal.

You might be correct on this. I m not so great in SEO anyways. I will keep your advice. 
I thought if Google search engine bots gets to know that this image is related to this site, it might help out in SEO.